sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0-beta.6 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
```

Comment:
Please add this section to `1.0.0-beta.5` and delete the changelog of `1.0.0-beta.6`.